Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,14 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
```

Comment:
What if this is a PBP, like an official demo or PSN game?  It seems like this would still run?

-[Unknown]